.github/CODEOWNERS

```diff
@@ -214,9 +214,8 @@
 /x-pack/plugins/telemetry_collection_xpack/ @elastic/kibana-core
 /.telemetryrc.json @elastic/kibana-core
 /x-pack/.telemetryrc.json @elastic/kibana-core
-src/plugins/telemetry/schema/legacy_oss_plugins.json @elastic/kibana-core
-src/plugins/telemetry/schema/oss_plugins.json @elastic/kibana-core
-x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json @elastic/kibana-core
+src/plugins/telemetry/schema/ @elastic/kibana-core @thesmallestduck @elastic/infra-telemetry
+x-pack/plugins/telemetry_collection_xpack/schema/ @elastic/kibana-core @thesmallestduck @elastic/infra-telemetry
```

Comment:
Q: Should we replace the `@thesmallestduck` mention to a team instead? Maybe we want to repurpose `@elastic/kibana-telemetry`?

Reply:
I'm generally +1 to using teams over individuals, to make maintenance easier. Not sure what the right team would be, though 🤔 

Would it make sense to add @thesmallestduck to `kibana-core`, perhaps? Or maybe a `kibana-core-pm` group, if there is one?